src/main/cpp/t08_chess.cpp

```diff
@@ -19,11 +19,65 @@
 
 #include "t08_chess.h"
 #include <iostream>
-
+#include <iomanip>
 
 using namespace std;
 
 int t08_chess() {
-    
-    return 0;
+
+
+	int a[100][100];
+	int n;
+	int m;
+	cin >> n >> m;
+	int count = 1;
+	int numb = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'numb' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951809)